sqlstore.go

```diff
@@ -48,23 +54,44 @@ func (f sqlStoreFactory) Create(sessionID SessionID) (msgStore MessageStore, err
 			return nil, err
 		}
 	}
-	return newSQLStore(sessionID, sqlDriver, sqlDataSourceName, sqlConnMaxLifetime)
+	sqlConnMaxIdle := 100
```

Comment:
Using default values from OMS

Reply:
this is too much for gotrader. 10/20 for idle/max should be good enough default.

Reply:
Lowered it to 10. This is more for a default value if you omit passing the settings. I also updated gotrader to pass a value when creating a new fix client

Reply:
Can you also update the max open conns?

Reply:
Yes, done